server/re/src/genQueryMS.cpp

```diff
@@ -347,7 +347,6 @@ msiGetMoreRows( msParam_t *genQueryInp_msp, msParam_t *genQueryOut_msp, msParam_
 
     if ( rei->status == 0 || rei->status == CAT_NO_ROWS_FOUND) {
         /* return continuation index separately in case it is needed in conditional expressions */
-        resetMsParam( continueInx );
```

Comment:
Looking into this as the `inpOutBuf` is being free'd here as well. If the pointers are set to `nullptr` after free'ing, the issues I found should be prevented as well as the leaks

Reply:
that sounds correct.   can't leak a nullptr, so I've heard.